systemd/systemd_test.go

```diff
@@ -113,6 +113,18 @@ func (s *SystemdTestSuite) TearDownTest(c *C) {
 	s.restoreSELinux()
 }
 
+// SystemctlError is returned if the mocked systemctl fails
+type SystemctlError struct {
```

Comment:
Although the Stop() function does not rely on a specific error type, but there's already an `Error` type which carries both the exit status and the output. IIRC only `is-active` and `is-enabled` rely on this error, as systemctl exits with non-0 status and prints out `inactive` to stdout`. So in order to distinguish a case where systemctl fails because of some random error, and operates successfuly but the service is inactive/disable we need to inspect bo the exit code and the output. Alternatively, you can just use errors.New(...) in places where this level of detail is not needed.

Reply:
Removed as per suggestion.